src/tilemaker/entrypoint.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+
+# Downloads a shapefile and converts it to raster tiles that it writes to S3
+
+set -e
+
+# Get the job ID either as an arg or from the environment
+PFB_JOB_ID="${1:-$PFB_JOB_ID}"
+
+if [ -z "${PFB_JOB_ID}" ]; then
+    echo "Error: PFB_JOB_ID is required"
+    exit 1
+fi
+
+TL_SHAPEFILE_NAME="${TL_SHAPEFILE_NAME:-neighborhood_ways}"
+TL_MIN_ZOOM="${TL_MIN_ZOOM:-8}"
+TL_MAX_ZOOM="${TL_MAX_ZOOM:-17}"
+
+PFB_TEMPDIR=`mktemp -d`
+
+TL_AWS_RESULTS_PATH="s3://${AWS_STORAGE_BUCKET_NAME}/results/${PFB_JOB_ID}"
```

Comment:
It might be better if this was passed in as part of the ENV? The AnalysisJob already manages the path a job's files exist at, and keeping AnalysisJob as the single source of truth for that is probably good, rather than having it defined here in a second spot.